build.gradle.kts

```diff
@@ -62,7 +62,6 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-data-jpa")
   implementation("org.springframework.boot:spring-boot-starter-validation")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:$hmppsSqsVersion")
```

Comment:
Incredibly picky and minor, and I hate myself for mentioning it .... (my inner "Kennard" is nagging at me! 😁 ), but is the var `hmppsSqsVersion` used anywhere else? Can we remove it?
(I told you it was incredibly nit-picky! - sorry!)